Cython/Compiler/Builtin.py

```diff
@@ -160,6 +160,9 @@ def declare_in_type(self, self_type):
     #('compile',   "",     "",      ""), # PyObject* Py_CompileString(    char *str, char *filename, int start)
     BuiltinFunction('delattr',    "OO",   "r",     "PyObject_DelAttr"),
     BuiltinFunction('dir',        "O",    "O",     "PyObject_Dir"),
+    BuiltinFunction('divmod',     "ii",   "O",     "__Pyx_divmod_int",
+                    utility_code=UtilityCode.load("divmod_int", "Builtins.c"),
+                    is_strict_signature = True, nogil = True),
```

Comment:
`nogil=True` seems dodgy for something that returns a Python tuple.

Reply:
Removed in the second commit, please review. Thank you.